packages/twenty-server/src/modules/workflow/workflow-executor/utils/variable-resolver.util.ts

```diff
@@ -0,0 +1,76 @@
+import { isNil, isString } from '@nestjs/common/utils/shared.utils';
+
+import Handlebars from 'handlebars';
+
+import {
+  WorkflowExecutorException,
+  WorkflowExecutorExceptionCode,
+} from 'src/modules/workflow/workflow-executor/exceptions/workflow-executor.exception';
+
+const VARIABLE_PATTERN = RegExp('\\{\\{(.*?)\\}\\}', 'g');
+
+export const resolve = (
+  unresolvedInput: any,
```

Comment:
Is it possible for us to predict a more specific type for the `unresolvedInput` parameter? If it's not, I would use `unknown` to be forced to explicitly check the type of the arguments.

Reply:
nope as explained above. So using unknown!